Albania/ALB_export_excel_output.py

```diff
@@ -0,0 +1,349 @@
+# Databricks notebook source
+# MAGIC %run ../utils
+
+# COMMAND ----------
+
+# MAGIC %pip install xlsxwriter
+
+# COMMAND ----------
+
+import tempfile
+import shutil
+
+from pyspark.sql.functions import col, sum as _sum, lit
+from pyspark.sql import functions as F
+
+import pandas as pd
+from openpyxl import load_workbook
+
+import logging
+logger = logging.getLogger()  # Get the root logger
+logging.getLogger("py4j").setLevel(logging.WARNING)
+logger.setLevel(logging.INFO)
+
+APPLY_BLUE_FONT_IF_MISSING = False
+
+OUTPUT_DIR = f"{TOP_DIR}/Workspace/output_excel"
+AUXI_DIR = f"{TOP_DIR}/Documents/input/Auxiliary"
+INPUT_AUXI_DIR = f"{TOP_DIR}/Documents/input/Auxiliary"
+
+OUTPUT_FILE_PATH = f"{OUTPUT_DIR}/Albania_BOOST.xlsx"
+SOURCE_FILE_PATH = f"{INPUT_DIR}/Albania BOOST.xlsx"
+TARGET_TABLE = 'prd_mega.boost.alb_publish'
+
+
+
+# COMMAND ----------
+
+
+# Load and filter
+raw_data = (
+    spark.table(f"{TARGET_TABLE}")
+    .withColumn("year", col("year").cast("string"))
+    .cache()
+)
+
+required_columns = [
+    "boost_admin0",
+    "boost_admin1",
+    "boost_admin2",
+    "boost_econ",
+    "boost_econ_sub",
+    "boost_func",
+    "boost_func_sub",
+    "boost_is_foreign",
+    "boost_approved",
+    "boost_executed",
+    "boost_year",
+]
+
+for col_name in required_columns:
+    if col_name not in raw_data.columns:
+        raise KeyError(f"Required column '{col_name}' is missing in the DataFrame.")
+
+tag_code_mapping = pd.read_csv(f"{AUXI_DIR}/tag_label_mapping.csv") 
+years = [str(year) for year in sorted(raw_data.select("year").distinct().rdd.flatMap(lambda x: x).collect())]
+
+def create_pivot(df, parent, child, agg_col ):
+    filtered_mapping = tag_code_mapping[tag_code_mapping['subnational'].isna()]
```

Comment:
Can we do without the "subnational" column and instead look for "_SBN_" in the code column? @yukinko-iwasaki